test/components/Modal.test.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Modal component and subcomponent test file
+ * Tests the fullscreen modal component for functionality and content
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import { ModalRaw as Modal } from '../../client/src/components/modal/Modal.jsx';
+import { PhotoReelRaw as PhotoReel } from '../../client/src/components/modal/PhotoReel.jsx';
+import {
+  Close,
+  Next,
+  Prev,
+  PhotoNav,
+} from '../../client/src/components/modal/Buttons.jsx';
+
+import testPhotoData from '../../data/testPhotoData';
+
+describe('buttons', () => {
```

Comment:
@krise3 not sure if this first describe is necessary considering you describe each button individually later on.  However it does clarify that there will be multiple button tests